addons.yaml

```diff
@@ -181,3 +181,11 @@ microk8s-addons:
       confinement: "classic"
       supported_architectures:
         - amd64
+
+    - name: "parking"
+      description: "Static webserver to park a domain. Works with EasyHAProxy."
+      version: "latest"
```

Comment:
@byjg could we please set a specific version? See comment https://github.com/canonical/microk8s-community-addons/pull/133#discussion_r1114547462